menus/language-arma-atom.cson

```diff
@@ -0,0 +1,11 @@
+'menu': [{
+  label: 'Packages'
+  submenu: [
+    label: 'Language Arma Atom',
+    submenu: [
+        { label: 'build', command: 'language-arma-atom:build' },
+        { label: 'make', command: 'language-arma-atom:make' },
+        { label: 'open RPT', command: 'language-arma-atom:open-rpt-file' },
```

Comment:
`label` here should be upper-cased, like that (this is the menu items):
`Build`
`Make`
`Open RPT`